lib/fb/metric.rb

```diff
@@ -0,0 +1,40 @@
+# Ruby client to authenticate a Facebook user.
+# @see http://www.rubydoc.info/gems/Fb/
+module Fb
+  # Fb::Metric reprensents a Facebook insights metric. Metric provides getters
+  # for: :name, :period, :title, :description, :id, and :values.
+  class Metric < Resource
```

Comment:
I find that I need to use symbolize_keys in many places and they're not necessarily a `resource`. I think we could  make a helper module instead.